pkg/consensus/payload.go

```diff
@@ -224,15 +230,39 @@ func (p *Payload) DecodeBinaryUnsigned(r *io.BinReader) {
 	}
 }
 
+// GetSignedHash returns a hash of the payload used to verify it.
+func (p *Payload) GetSignedHash() util.Uint256 {
+	if p.signedHash.Equals(util.Uint256{}) {
+		p.createHash()
```

Comment:
Why do we ignore error here? (vs panic in `Hash()`)

Reply:
Fixed.